botocore/httpchecksum.py

```diff
@@ -250,7 +250,14 @@ def resolve_request_checksum_algorithm(
             supported_algorithms = _SUPPORTED_CHECKSUM_ALGORITHMS
 
         algorithm_name = params[algorithm_member].lower()
-        if algorithm_name not in supported_algorithms:
+        if algorithm_name == "crc32c" and not HAS_CRT:
+            raise FlexibleChecksumError(
+                error_msg=(
+                    "CRC32C is not supported without the CRT. To use "
+                    "CRC32C, please install the `awscrt` package."
```

Comment:
Should we explicitly say the command here? `pip install awscrt`

Reply:
In other places we've used messages such as:
```python
"Using S3 with an MRAP arn requires an additional "
"dependency. You will need to pip install "
"botocore[crt] before proceeding."
```
Maybe we can do something similar here?
```python
"Using CRC32C requires an additional "
"dependency. You will need to pip install "
"botocore[crt] before proceeding."
```
Source: https://github.com/boto/botocore/blob/develop/botocore/utils.py#LL2206C17-L2210C18

Reply:
Im not extremely familiar with using CRT but it might be safer to have users install CRT through `botocore[crt]` since this ensures they install a compatible version. We're currently pinning to `awscrt==0.16.9`.

https://github.com/boto/botocore/blob/develop/setup.cfg#LL11C1-L11C21

Reply:
Yep you're right on both counts. I'll update shortly.